src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,26 @@
+package racingcar.domain
+
+data class Cars(private val cars: List<Car>) {
+    fun move(condition: () -> Int): List<Car> {
+        val of = mutableListOf<Car>()
+
+        cars.forEach {
+            of.add(it.move(condition()))
+        }
+        return of
+    }
+
+    fun findMaxStatusValue(): Int = cars.maxOf { it.status }
+
+    fun statistic(max: Int): List<Car> {
+        val result = mutableListOf<Car>()
+        cars.forEach { car ->
+            run {
+                if (car.isWinner(max)) {
+                    result += car
+                }
+            }
+        }
+        return result
+    }
```

Comment:
```suggestion
    fun statistic(max: Int): List<Car> {
        return cars.filter { it.isWinner(max) }
    }
```

`filter`를 사용하면 더 간결해지지 않을까요?

Reply:
@Flamme1004K 반영했습니다!